Packs/ApiModules/Scripts/ServiceNowApiModule/ServiceNowApiModule.yml

```diff
@@ -12,5 +12,7 @@ comment: Common ServiceNow code that will be appended to each ServiceNow integra
 scripttarget: 0
 dependson: {}
 timeout: 0s
-dockerimage: demisto/python3:3.8.6.13358
+dockerimage: demisto/crypto:1.0.0.2005673
```

Comment:
note: 
I choose this docker image since the[ build could not import jwt using demisto/auth-utils](https://github.com/demisto/content/actions/runs/13358408253/job/37304372847):

`_ ERROR collecting Packs/ApiModules/Scripts/ServiceNowApiModule/ServiceNowApiModule_test.py _
ImportError while importing test module '/src/Packs/ApiModules/Scripts/ServiceNowApiModule/ServiceNowApiModule_test.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/local/lib/python3.11/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
ServiceNowApiModule_test.py:1: in <module>
    from ServiceNowApiModule import *
ServiceNowApiModule.py:3: in <module>
    import jwt
E   ModuleNotFoundError: No module named 'jwt'`